src/main/java/org/robotframework/remoteserver/library/DynamicApiRemoteLibrary.java

```diff
@@ -41,7 +41,8 @@ protected DynamicApiRemoteLibrary(Object library, Method getKeywordNames, Method
         try {
             Object names = getKeywordNames.invoke(library, new Object[] {});
             if (names instanceof List) {
-                return (String[]) ((List<?>) names).toArray();
+                Object[] nameArray = ((List<?>) names).toArray();
```

Comment:
I think that perhaps just using the other toArray would fix this.
`return (String[]) ((List<?>) names).toArray(new String[0]);`

Reply:
This worked, the cast to `String[]` can also be omitted as toArray already returns a string array.
`return ((List<?>) names).toArray(new String[0]);`
Commit https://github.com/ombre42/jrobotremoteserver/pull/43/commits/c6714e782afac82a061093f6a8f1b8984f4ef8a0